src/features/landing/components/ForWhoSection/styles.ts

```
@@ -1,5 +1,34 @@
+const bp = breakpointsTheme.breakpoints.values;
+
+
+const mq = {
+  max968: '@media (max-width: 968px)',
+  max1131: '@media (max-width: 1130.98px)',
+  min426: '@media (min-width: 426px)',
+  min468: '@media (min-width: 468px)',
+  min641: '@media (min-width: 641px)',
+  min969: '@media (min-width: 969px)',
+  min1131: '@media (min-width: 1131px)',
+};
```

ESLint

[error] Expected mq to have a type annotation.

(@typescript-eslint/typedef)